### PR TITLE
Update define-a-docker-build-infrastructure.md

### PR DESCRIPTION
**Pull request by nofarblue**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [X] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
 ---
-title: Define a Docker Build Infrastructure
-description: You can define a CI build infrastructure on any Linux or MacOS host. This is the simplest build infrastructure to set up and is well suited to developers who want to build on their laptops.
+title: Define a Local Build Infrastructure
+description: You can define a CI build infrastructure on Linux, Windows and MacOS machines. This is the simplest way to run self-hosted builds.
```
Comment (nofarblue) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:3]: need to be macOS instead of MacOS

Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:3]:
```suggestion
description: You can define a CI build infrastructure on Linux, Windows and macOS machines. This is the simplest way to run self-hosted builds.
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 sidebar_position: 15
 helpdocs_topic_id: xd8u17be5h
 helpdocs_category_id: rg8mrhqm95
 helpdocs_is_private: false
 helpdocs_is_published: true
 ---
 
-You can define a CI build infrastructure on any Linux or MacOS host. This is the simplest build infrastructure to set up, and is well-suited for developers who want to run builds on a local host such as a laptop.
+With the Local runner (AKA Docker runner) you can run builds on self-hosted Linux, Windows and MacOS machines. It allows you to re-use your machines to run your builds on. You can choose whether to execute build steps directly on the host machine or in Docker containers (if docker engine is avaiable).
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:11]:
```suggestion
With the Local runner (also known as the Docker runner) you can run builds on self-hosted Linux, Windows, and macOS machines. This allows you to re-use machines that you run builds on. If Docker Engine is available, you can choose whether to execute build steps directly on the host machine or in Docker containers.
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+
+
 
 ### Important Notes
 
+* Setting up a local build environment requires running a docker compose file that installs a Harness delegate and the Local runner
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:17]:
```suggestion
* To set up a local build environment, you must run a Docker Compose file that installs a Harness Delegate and the Local runner.
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
 
 ### Important Notes
 
+* Setting up a local build environment requires running a docker compose file that installs a Harness delegate and the Local runner  
 * The Docker Delegate has the following system requirements:
 	+ Default 0.5 CPU.
 	+ Default 1.5GB. Ensure that you provide the minimum memory for the Delegate and enough memory for the host/node system.
-* You need to download and install a Drone runner in addition to the delegate.
-* The Docker delegate is bound by the total amount of memory and CPU on the local host. If you run multiple builds and the host runs out of CPU or memory, some builds will time out and fail.
+* The Docker delegate is bound by the total amount of memory and CPU on the local host. If you run multiple builds and the host runs out of CPU or memory,  builds could time out and fail.
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:21]:
```suggestion
* The Docker Delegate is limited by the total amount of memory and CPU on the local host. Builds can fail if the host runs out of CPU or memory when running multiple builds.
```

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md (hunk 1 above)
Comment (michaelcretzman) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:3]:
```suggestion
title: Define a local build infrastructure
description: Define a CI build infra on local Linux/Windows/MacOS machine.
```

Comment (michaelcretzman) [reply docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:3]: title must follow sentence case per style guide: https://harness.atlassian.net/wiki/spaces/DOCS/pages/21130772574/Documentation+style+guide

Description should be short as the card that displays it is very small and any text longer than a few words is cut off. You can see it here: https://developer.harness.io/docs/category/set-up-build-infrastructure

**Review by michaelcretzman — DISMISSED** [review]
Approving with the assumption that you'll consider my suggestions.

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 5]
```diff
@@ -1,22 +1,24 @@
 ---
-title: Define a Docker Build Infrastructure
-description: You can define a CI build infrastructure on any Linux or MacOS host. This is the simplest build infrastructure to set up and is well suited to developers who want to build on their laptops.
+title: Define a Local Build Infrastructure
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:2]:
```suggestion
title: Define a local build infrastructure
description: Define a CI build infra on local Linux/Windows/MacOS machine.
```

**Review by aimurphy — APPROVED** [review]

File: docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md [hunk 6]
```diff
@@ -1,22 +1,25 @@
 ---
-title: Define a Docker Build Infrastructure
-description: You can define a CI build infrastructure on any Linux or MacOS host. This is the simplest build infrastructure to set up and is well suited to developers who want to build on their laptops.
+title: Define a local build infrastructure
+description: Define a CI build infra on local Linux/Windows/MacOS machine.
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/set-up-build-infrastructure/define-a-docker-build-infrastructure.md:3]:
```suggestion
description: Define a CI build infra on local Linux/Windows/macOS machine.
```

**Review by aimurphy — APPROVED** [review]

**Review by michaelcretzman — APPROVED** [review]

**Pull request merged**